docs/BR.md

```diff
@@ -2089,10 +2091,10 @@ Table: Policy Restricted
 
 Table: Permitted `policyQualifiers`
 
-| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| __policyQualifier__                     | __Presence__ | __Field Type__ |  __Contents__ |
 | ---                                  | -            | -              | -----         |
-| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | The HTTP or HTTPS URL for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
-| Any other qualifier                  | MUST NOT     | -              | -             |
+| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | When provided, every policyQualifier SHALL have the uniformResourceIdentifier SCHEME HTTP or HTTPS for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
```

Comment:
I don't think `SCHEME` needs to be capitalized here; it should be `Scheme` instead, as a defined term.
The phrasing here can also be simplified, imo, to `The uniformResourceIdentifier using the HTTP or HTTPS Scheme for the....`
If keeping the current wording, I think it should be updated to `When provided, the id-qt-cps policyQualifierId SHALL have the uniformResourceIdentifier Scheme HTTP or HTTPS for the....`

Reply:
Sounds good.